query-api/src/test/java/com/brandwatch/minibcr/queryapi/repository/UserQueryRepositoryTest.java

```diff
@@ -0,0 +1,32 @@
+package com.brandwatch.minibcr.queryapi.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.brandwatch.minibcr.queryapi.config.PostgresContainerTest;
+import com.brandwatch.minibcr.queryapi.entity.UserQuery;
+
+@SpringBootTest
+public class UserQueryRepositoryTest extends PostgresContainerTest {
```

Comment:
It looks like you need Postgres and Redis, so I think what you'd want to do instead of extending is having an initaliser in this class that starts both containers and passes the properties? Or maybe have the initializer in a dedicated class 🤷 

There are certainly several ways to work with testcontainers